xhprof_lib/templates/profChart.phtml

```diff
@@ -68,7 +68,7 @@
                     $other+= $dataPoint['excl_wt'];
                 }else
                 {
-                    echo "['{$dataPoint['fn']}', {$dataPoint['excl_wt']}],\n";
+                    echo "['" . addslashes($dataPoint['fn']) . "', {$dataPoint['excl_wt']}],\n";
```

Comment:
The `addslashes` is not the method designed to transform PHP data type into JS data type. You should be using `json_encode` instead.